libkbfs/folder_block_ops.go

```diff
@@ -2588,6 +2586,7 @@ func (fbo *folderBlockOps) getDeferredWriteCountForTest(lState *lockState) int {
 }
 
 func (fbo *folderBlockOps) updatePointer(kmd KeyMetadata, oldPtr BlockPointer, newPtr BlockPointer) {
+	fbo.log.CDebugf(context.Background(), "Updating reference for pointer %s to %s", oldPtr.ID, newPtr.ID)
```

Comment:
This is going to be a pretty intensely frequent log message, and only useful in certain conditions.  But there have been times when I've want something like it.  If this is intentional, please print out the full `BlockPointer`, instead of just the ID, since the refnonces could be useful as well.